tests/libraries/test_sortedLinkedList.py

```diff
@@ -19,21 +19,21 @@ def sortedLinkedListContractsFixture(sessionFixture, sortedLinkedListSnapshot):
     sessionFixture.chain.revert(sortedLinkedListSnapshot)
     return sessionFixture
 
-def test_helperComparor(sortedLinkedListContractsFixture):
-
-    int256Comparor = sortedLinkedListContractsFixture.contracts['int256Comparor']
-
-    assert int256Comparor.compare(1, 1) == 0
-    assert int256Comparor.compare(0, 1) == -1
-    assert int256Comparor.compare(1, 0) == 1
+#def test_helperComparor(sortedLinkedListContractsFixture):
+#
+#    int256Comparor = sortedLinkedListContractsFixture.contracts['int256Comparor']
+#
+#    assert int256Comparor.compare(1, 1) == 0
+#    assert int256Comparor.compare(0, 1) == -1
+#    assert int256Comparor.compare(1, 0) == 1
```

Comment:
These should be uncommonted or deleted (deleted if you remove Comparor contract).